added feature block

shewood opened the pull request:
## [DC-50](https://jira-dev.bdm-dev.dts-stn.com/browse/DC-50) (Jira Issue)

### Description
Added feature block section with text and image. 
updated action button link part to have default styles and also custom styles. 

List of proposed changes:

-
-

### What to test for/How to test

### Addtional Notes

nibivi77 commented:
Dev branch:
![image](https://user-images.githubusercontent.com/77116011/136610828-bfd26f44-533a-4a00-8d9b-e372681a5d2f.png)


With the new ActionButton:
![image](https://user-images.githubusercontent.com/77116011/136610917-7585cc96-c0a4-46d2-a936-fe709b463284.png)

MarcoGoC commented on components/atoms/ActionButton.js:
```diff
@@ -17,12 +18,13 @@ export function ActionButton({
       <a
-        className={`${!className ? defaultStyle : className}`}
+        className={`${defaultStyle} 
+        ${extendedClass && !className ? extendedClass : ''}`}
```
This line does not do what you expect. className is never executed

shewood replied:
Please help me fix it.

MarcoGoC reviewed (changes requested):
See comments

shewood commented on components/molecules/FeatureBlock.js:
```diff
@@ -0,0 +1,59 @@
+        <p className="font-body my-5">{props.body}</p>
+        <div className=" ">
+          <ActionButton
+            defaultstyle="true"
```
I removed the action button changes.

nibivi77 commented on components/atoms/ActionButton.js:
```diff
@@ -7,6 +7,7 @@ import Link from 'next/link'
 export function ActionButton({
   href,
   className,
+  extendedClass,
```
Since we are not using it, we can remove the extendedClass in both places (line 10 and line 87)

shewood replied:
Oops, yes, I forgot those

nibivi77 commented on components/molecules/FeatureBlock.js:
```diff
@@ -0,0 +1,54 @@
+      id="feature-block"
+    >
+      <div className="bg-splash-page bg-cover bg-center w-full py-32  ">
+        {/* <div className="aspect-w-16 aspect-h-9">
```
I assume this is a placeholder? Do we need it?

shewood commented:
> Looking gooood! Quick feedback: Our branch looks like this: ![image](https://user-images.githubusercontent.com/77116011/137201214-83d7f8a7-4e03-449f-a09e-7b242d60d3e3.png)
> 
> And the Figma looks like this: ![image](https://user-images.githubusercontent.com/77116011/137201278-1e973605-cf09-4c8b-a349-f7bc352d15cb.png)
> 
> The button seems to be a little bit off.

<cry> this is why I updated the button so I could style it as I wanted but I added a wrapper and maybe I can style it there instead.
